src/api/index.js

```diff
@@ -5,9 +5,12 @@ import _axios, { create } from 'axios';
 import { pathOr, equals, compose } from 'ramda';
 import qs from 'qs';
 
+const baseURL = process.env && process.env.NODE_ENV === 'development'
```

Comment:
I feel like you mentioned something about this but don't remember why we've gone back to doing this instead of using `REACT_APP_` env vars?